srfi-0.html

```diff
@@ -1,3 +1,9 @@
+<!--
+SPDX-FileCopyrightText: 1999 Marc Feeley
+
+SPDX-License-Identifier: MIT
```

Comment:
This is not the license listed at the bottom of the document.  Note that the editors at the time hadn't yet settled on the MIT license.  The license used in this SRFI also appears in SRFIs 0, 5, 40, 44, 54, 57, 58, and 60.